Makefile

```diff
@@ -4,20 +4,21 @@
 
 SENTRY_ORG=testorg-az
 SENTRY_PROJECT=ng-demo
-VERSION=`sentry-cli releases propose-version`
 PREFIX=dist
+SENTRY_CLI=./node_modules/.bin/sentry-cli
+VERSION=`$(SENTRY_CLI) releases propose-version`
 
 setup_release: create_release associate_commits upload_sourcemaps
 
 create_release:
-	sentry-cli releases -o $(SENTRY_ORG) new -p $(SENTRY_PROJECT) $(VERSION)
+	$(SENTRY_CLI) releases -o $(SENTRY_ORG) new -p $(SENTRY_PROJECT) $(VERSION)
 
 associate_commits:
-	sentry-cli releases -o $(SENTRY_ORG) -p $(SENTRY_PROJECT) set-commits --auto $(VERSION)
+	-$(SENTRY_CLI) releases -o $(SENTRY_ORG) -p $(SENTRY_PROJECT) set-commits --local $(VERSION) 
 
 upload_sourcemaps:
-	sentry-cli releases -o $(SENTRY_ORG) -p $(SENTRY_PROJECT) files \
+	$(SENTRY_CLI) releases -o $(SENTRY_ORG) -p $(SENTRY_PROJECT) files \
 		$(VERSION) upload-sourcemaps --url-prefix "~/" --rewrite --validate $(PREFIX)
 
-reference_release:
-	sed -i -e "s/release: .*/\release: \"${VERSION}\"/g" src/app/app.module.ts
+create_env:
+	@echo "version=${VERSION}" >> .env
```

Comment:
it's becasue of this.
should be `>` instead of `>>`

Reply:
@ndmanvar ok updated, merging now